validation/report.md

```diff
@@ -4,23 +4,35 @@ This report summarizes the Phase II validation efforts by the global indicators
  
-A framework containing more than 25 methods and indicators is presented, allowing OSM quality assessments based solely on the data’s history (OSM-Full-History-Dump). In lieu of a reference dataset, approximate statements on OSM data quality are possible. No ground truth reference dataset is deployed for OSM data quality evaluation. For example, an intrinsic analysis approach is applied in specified areas within OSM that can be evaluated by investigating the data’s historical development and comparing features’ characteristics at different timestamps. In order to assess the OSM data, the calculated results of the iOSMAnalyzer are divided into the following categories: fitness for purpose; general information on the study area; routing and navigation; geocoding; points of interest-search; map-applications; and user information and behavior. The calculated results give a compact quality overview of a freely selectable area. Quality depends on the individual use case, and the OSM data is evaluated in terms of fitness for purpose. However, absolute statements on data quality are only possible with a high-quality reference dataset as a basis for comparison. The study revealed that the interpretation of quality indicators is facilitated and supported by means of contributor activity (Barron et al., 2014). 
+As a result, a framework containing more than 25 methods and indicators is presented. This framework allows OSM quality assessments based solely on the data’s history (OSM-Full- History-Dump). In lieu of a reference dataset, approximate statements on OSM data quality are possible. No ground truth reference dataset is deployed for OSM data quality evaluation. Instead, an alternate approach is used in specified areas within OSM. This approach can be evaluated by investigating the data’s historical development and comparing features’ characteristics at different timestamps. In order to assess the OSM data, the calculated results of the iOSMAnalyzer are divided into the following categories: fitness for purpose; general information on the study area; routing and navigation; geocoding; points of interest-search; map-applications; and user information and behavior. The calculated results give a compact quality overview of a freely selectable area. Quality depends on the individual use case, and the OSM data is evaluated in terms of fitness for purpose. However, absolute statements on data quality are only possible with a high-quality reference dataset as a basis for comparison. The study revealed that the interpretation of quality indicators is facilitated and supported by means of contributor activity (Barron et al., 2014). 
```

Comment:
Can you do a find/replace-all for the ’ character and replace it to the standard ' apostrophe?